contracts/VetoERC20Voting.sol

```diff
@@ -0,0 +1,181 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./interfaces/IERC20Votes.sol";
+import "./interfaces/IVetoGuard.sol";
+import "./interfaces/IVetoERC20Voting.sol";
+import "./TransactionHasher.sol";
+import "@fractal-framework/core-contracts/contracts/ModuleBase.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+/// @notice A contract for casting veto votes with an ERC20 votes token
+contract VetoERC20Voting is IVetoERC20Voting, TransactionHasher, ModuleBase {
+    uint256 public vetoVotesThreshold;
```

Comment:
Oh yeah that's a great point. Make this value a percentage of the votes tokens current supply. I'll look into the best way to implement this.

Reply:
Yea - perhaps we just snag it from the quorum function on the votes token and just use the same mechanism here - read from the contract instead of maintaining the state here. thoughts? should it be separate - I see it going both ways.

Reply:
What's the quorum function on the votes token you're referring to?

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/extensions/GovernorVotesQuorumFraction.sol

Reply:
Right, its on the Governor not the votes token